### DISCUSSION TITLE
Sync between devices (Apple like) for better UX

### DISCUSSION THREAD

**Discussion by kirillkurko** [dailydotdev/daily#704] [Feature Requests]
For example, Apple shows you a bottom drawer with a suggestion to share your Wi-Fi password, so you don't need to type in on a new device.

For now, it feels like the most comfortable platform is the desktop because you can do everything here with good UX (reading, writing comments etc.), but other platforms are not that good (I found, that reading experience on iPad is great, but it's terrible for writing comments). So maybe platform can have sync between devices, e.g. articles modal opened on smartphone should be also opened on desktop etc. This idea required more real-time behavior, and one more possible way is to have a popup saying "open in on another device".

The idea is row, and I wasn't thinking much about actual implementation and possible technical limitations. Also, the whole feature is not that important if the majority of users are on desktop. But such interactions can improve UX on phones and tablets and engage people to start using new devices

**Comment by idoshamun** [comment]
Hi @kirillkurko,

That's a very interesting and cool idea! Once we will start exploring the mobile experience (which indeed needs quite work), we will look at devices sync as you suggest.
